orchagent/dash/dashvnetorch.cpp

```diff
@@ -310,7 +325,7 @@ bool DashVnetOrch::addOutboundCaToPa(const string& key, VnetMapBulkContext& ctxt
             else
             {
                 SWSS_LOG_ERROR("Invalid encap type %d for %s", action.encap_type(), key.c_str());
-                return false;
+                return true;
```

Comment:
why true?

Reply:
The `true` means "error", so it will be removed from m_toSync.
While `false` means "ok, proceed" or "wait for another resource that is not yet available", not removing from the m_toSync.

https://github.com/sonic-net/sonic-swss/blob/1847195cf02a2baa478a664a671adc95c704a810/orchagent/dash/dashvnetorch.cpp#L737-L744